### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_EditMetada_ScreenReader: Narrator does not read the 'Code' written in the code block Properly.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4014]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106 
Operating System:Windows10

**Repro Steps:**
1.Login with the appropriate credentials to azure notebook account.    
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Navigate to run button & hit enter --> Navigate to Creation and Deployment of an Azure ML Web Service & hit enter.
4.Start Narrator.
5.For each cell, two modes are provided for using shortcut keys, i.e. 
* EditMode( Represented by Green Cell Border)
* CommandMode( Represented by Blue Cell Margin)
6. Verify if there is any other way to identify the current mode of the highlighted cell. 

**Actual:**
There is no other way to identify whether the current cell is in 'Edit' mode or 'Command' mode instead of the 'Color' representation for the cells.

**Expected:**
Any information that is provided with the help of colors should also be provided through other ways such as 'Screenreader' or 'Text' form.

**User Impact:**
Color blind user will not be able to identify the current mode of the highlighted cell i.e. whether the cell is in 'Edit' mode or 'Command' mode.

**Attachment:**
(https://github.com/jupyter/notebook/files/2401308/A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
